### PR TITLE
Android beta support

### LINKED ISSUE #4: Android support
I'm unable to get this working on Android. The view loads an empty Tableview.

I think this might be related to https://jira.appcelerator.org/browse/TIMOB-14285 where Android doesn't apply custom properties to proxy.

I have been able to get the table show by using the following in widget.js:
```
  $.table.setData(tableProp.sections);
  $.table.applyProperties(tableProp);
```
But I am unable to access all the event listeners that have been created.

### PR DESCRIPTION
**Pull request by HazemKhaled**
Adding beta Android support, UI working fine, but i've to test listener and HeaderView, also pickers ot touched yet

fix #4 

screen shot 2016-04-06 at 3 44 10 pm

### CODE REVIEW

**Comment by FokkeZB** [comment]
Thanks @HazemKhaled! Could you let me know when you have finished this up?

**Comment by HazemKhaled** [comment]
I'm working on a new project right now and using this widget, the plan to
fix only any bug I'll face.

I think better to make sure this commits will not break the iOS features,
at least now text field, sections, headerTitle and footerTitle working on
both iOS and Android, anything else still working fine on iOS and not
tested on Android

On Fri, Apr 8, 2016, 10:23 AM Fokke Zandbergen <notifications@github.com>
wrote:

> Thanks @HazemKhaled <https://github.com/HazemKhaled>! Could you let me
> know when you have finished this up?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FokkeZB/nl.fokkezb.form/pull/18#issuecomment-207308222>
>
-- 
*Regards,*
Hazem Khaled | Software Engineer | About Me ...
<http://about.me/HazemKhaled>
Mob: +20 122 7660 187
         +971 52 815 6993

File: styles/field.tss [hunk 1]
```diff
@@ -1,26 +1,29 @@
 ".row[platform=ios]": {
-  selectionStyle: Ti.UI.iPhone.TableViewCellSelectionStyle.NONE,
-
-  layout: 'horizontal'
```
Comment (FokkeZB) [inline styles/field.tss]: Why remove horizontal layout?

Comment (HazemKhaled) [inline styles/field.tss]: TextField width on Android filling the row, don't know why, not layout is composite, #control left now is 160

**Comment by HazemKhaled** [comment]
HeaderView and FooterView working fine on both iOS and Android

**Pull request merged**